src/admin/admin.controller.ts

```diff
@@ -35,11 +35,22 @@ import { Customer } from 'src/user/services/spider/dto/spider.dto';
 import { SpiderApiService } from 'src/user/services/spider/spider-api.service';
 import { SpiderService } from 'src/user/services/spider/spider.service';
 import { getConnection } from 'typeorm';
+import { dbQueryDto } from './dto/db-query.dto';
 import { RenameRefDto } from './dto/rename-ref.dto';
 import { SendLetterDto } from './dto/send-letter.dto';
 import { SendMailDto } from './dto/send-mail.dto';
 import { UploadFileDto } from './dto/upload-file.dto';
 
+export interface dbQueryInput {
+  table: string;
+  min?: string;
+  updatedSince: string;
+  extended: string;
+  maxLine: string;
+  sorting: 'ASC' | 'DESC';
+  customCols?: string[];
+}
+
```

Comment:
Did you test the code? 😉  I nowhere see a mapping from filterCols (in DTO) to customCols in interface.

Reply:
Yeah Code is tested and working. The interface is just old unused code. I will remove it :+1: